superset/connectors/sqla/utils.py

```diff
@@ -231,7 +219,7 @@ def load_or_create_tables(  # pylint: disable=too-many-arguments
                     name=column["name"],
                     type=str(column["type"]),
                     expression=conditional_quote(column["name"]),
-                    is_temporal=is_column_type_temporal(column["type"]),
+                    is_temporal=column["is_dttm"],
```

Comment:
The previous version checked for `date`, `datetime`, `time` and `timedelta`.  Is this equivalent?

Reply:
yes - this one is actually based on what the db engine spec says is temporal, so it's much more comprehensive (=can be customized per database).